.github/workflows/scripts/scanner-offline-bundle-generate.sh

```diff
@@ -12,16 +12,20 @@ mkdir -p "$output_dir"
 
 filename=vulnerabilities.zip
 filename_version="$vulnerability_version"
+bundle_prefix=v4-definitions-
```

Comment:
I was never a fan of the word definitions for this, but I can't currently think of anything better, and I won't block over this

Reply:
We are roughly referring to the zip files embedded in the offline bundle as "definitions". I am just hitchhiking on the existence of this word, and the fact that the files have "defs" in them. Anyway, the name helps us sort out the different "bundles" we have, without revamping the terminology (we have the vulnerability bundle for v4 and v2, we have definition bundles, and we have the offline bundle).